admin/spec/models/check-window.spec.js

```diff
@@ -53,12 +68,100 @@ describe('check-window schema', function () {
     })
   })
 
-  it('requires a checkEndDate property', function (done) {
+  it('should require a checkEndDate property', function (done) {
     checkWindow.checkEndDate = undefined
     checkWindow.validate(error => {
       expect(error).toBeDefined()
       expect(error.errors.checkEndDate).toBeDefined()
       done()
     })
   })
+
+  describe('Method getCheckWindow', function () {
+    it('should return a check window by id', async function (done) {
+      expect(typeof CheckWindow.getCheckWindow).toBe('function')
+      sandbox
+        .mock(CheckWindow)
+        .expects('findOne')
+        .chain('exec')
+        .resolves(CheckWindowMock)
+
+      try {
+        const cw = await CheckWindow.getCheckWindow('5964fb80b42c79b8849b33f0')
+        const startDate = moment.utc('2017-08-26 00:00:01').toDate()
+        const checkStartDate = moment.utc('2017-09-10 00:00:01').toDate()
+        const checkEndDate = moment.utc('2017-09-12 00:00:01').toDate()
+
+        expect(cw.checkWindowName).toBe('Check Window Name')
+        expect(cw.startDate.toString()).toBe(startDate.toString())
+        expect(cw.checkStartDate.toString()).toBe(checkStartDate.toString())
+        expect(cw.checkEndDate.toString()).toBe(checkEndDate.toString())
+        expect(cw.isDeleted).toBe(false)
+        done()
+      } catch (err) {
+        console.log('ERROR', err)
+      }
+      done()
+    })
+
+    it('should fails', async function (done) {
```

Comment:
typo.  also, its not clear why it should fail, please add a condition to the description